src/docs/cmdstan-guide/running.tex

```diff
@@ -29,15 +29,15 @@ \subsubsection{Mac OS and Linux}
 %
 \begin{quote}
 \begin{Verbatim}[fontshape=sl]
-> ./my_model sample data file=my_data
+> ./my_model method=sample data file=my_data
```

Comment:
I think we've preferred using the form you erased.  They should do the same thing.  This is repeated throughout.

Reply:
done.